src/main.browser.ts

```diff
@@ -0,0 +1,8 @@
+import 'zone.js/dist/zone';
+import 'reflect-metadata';
+import 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { BrowserAppModule } from './app/browser-app.module';
+
+platformBrowserDynamic().bootstrapModule(BrowserAppModule);
```

Comment:
Wouldn't platformBrowser be a part of the virtual one that gets created when doing an AoT build? Since we're using ngtools here that is.